src/pybind/mgr/dashboard/controllers/saml2.py

```diff
@@ -17,7 +17,7 @@
 from ..exceptions import UserDoesNotExist
 from ..services.auth import JwtManager
 from ..tools import prepare_url_prefix
-from . import Controller, Endpoint, BaseController
+from . import BaseController, Controller, Endpoint, allow_empty_body, set_cookies
```

Comment:
No need to import `allow_empty_body` as it's not used anywhere in the file. Also this change comes from other PR.